Service/Microsoft.Health.DeIdentification.Local/LocalArtifactStore.cs

```diff
@@ -5,7 +5,9 @@ namespace Microsoft.Health.DeIdentification.Local
 {
     public class LocalArtifactStore : IArtifactStore
     {
-        readonly string _defaultConfigFile = "configurations/deid-configuration.json";
+
+        readonly string _defaultConfigFile = "../Microsoft.Health.DeIdentification.Local/configurations/deid-configuration.json";
```

Comment:
we can set "Copy To Output" of the deid-configuration.json file in project setting to true, so it will be always copied to output directory and we don't need change this variable.